### DISCUSSION TITLE
Add Image to Webhook

### DISCUSSION THREAD

**Discussion by wuast94** [grafana/grafana#71171] [Alerting]
Would be nice to have the image rendered by image rendering service included to the webhook. 

This could be just binary or base64 for example.

**Comment by yuri-tceretian** [comment]
Interesting idea, that might help users that do not want to configure publicly available storage for images to use the feature. 
I think it would be reasonable to let the user choose in what format the image should be included for each integration, i.e. via a new option in the webhook integration settings.

**Comment by armandgrillet** [comment]
Closing in favor of https://github.com/grafana/grafana/issues/64865 as we are now using issues for feature requests.
